### ISSUE TITLE
Login Key with azure SSO login fails to login

### ISSUE THREAD

**Issue by vish84** [Ylianst/MeshCentral#4910]
Hi

I managed to get Azure SSO working, but need to further restrict the web page login with a loginkey to restrict access to those that only have this secret key, but am struggling to make this work. 

If I enable the loginKey, I need to supply this to get to my URL, but am then unable to login with Azure SSO, and keep being looped back to the login screen after selecting my Azure AD account to login with. What am I missing? 
`
{
    "domains": {
	 "": {
 		"Title": "Remote Support",
 		"Title2": "Remote Support",
		 "loginKey":"SecretKey",
		 "showPasswordLogin": true,
		"newAccountsUserGroups": [ "ugrp//mvslrMPtegdFo2$Kore7$AM6fcUdOx7lJAjmmrvq1jh6ZYnL7WNygPy$GwL4I4lJ" ],
 		"NewAccounts": false,
    "authStrategies": {
 	"azure": {
		 "callbackurl": "https://domain.com/?key=SecretKey",
		"newAccounts": true,
		"clientid": "",
 		"clientsecret": "",
		"tenantid": ""
 		}
}

`

**Comment by whllm** [comment]
HA! A dead end! 

Did you ever manage to get this working? I've got the same setup, but I've set the callbackurl set as:

"callbackurl": "https://mesh.domain.tld/auth-azure-callback?key=1234"

This allows me to log in just fine, but after I get the token I'm redirected to / and the ?key=1234 is stripped off. I have to go up and re-enter the ?key manually to get to the UI. It works but it's gross and defeats the purpose of *seamless* sso.
